docs/12.Log-Viewing-Pages.md

```
@@ -0,0 +1,524 @@
+* **Next** (`>`) - Next page
+* **Last Page** (`>>`) - Jump to final page
+
+Pagination state is maintained through the `common_args` template variable [src/templates/logs.html L6-L13](https://github.com/ArWikiCats/ArWikiCatsWeb/blob/88f42d13/src/templates/logs.html#L6-L13)
+
+ which preserves all filter parameters across page navigation.
+
+Page information is displayed below the pagination controls [src/templates/logs.html L177-L180](https://github.com/ArWikiCats/ArWikiCatsWeb/blob/88f42d13/src/templates/logs.html#L177-L180)
+
+:
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)